tests/Integration/Behaviour/Features/Context/Domain/Product/UpdatePackFeatureContext.php

```diff
@@ -187,6 +208,29 @@ public function assertAddingPackToPackError()
         );
     }
 
+    /**
+     * @When I set following packed products to product :productReference:
+     *
+     * @param string $productReference
+     * @param TableNode $tableNode
+     */
+    public function setPackedProducts(string $productReference, TableNode $tableNode): void
```

Comment:
You don't need this method since there is already a `updateProductPack` that does the same thing
If you want to change the wording of the behat sentence you can assign two sentences to the same method, or you can even completely rephrase it and update its usage everywhere I don't mind Both sentences are fine for me

But only one method is needed

Reply:
This one still needs to be cleaned I believe, we don't need duplicate behat entries We only need the existing `updateProductPack` method But you can rename it if the naming doesn't seem adapted for you

Reply:
Fourth time, still no modif or at least an answer if I'm wrong...

Reply:
yes SetProductPack create a duplicate with this one. I have to wire the duplicate test on it